app/assets/stylesheets/components/profile-form.scss

```diff
@@ -0,0 +1,38 @@
+.profile-form {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  max-width: 700px;
+  position: relative;
+  width: 80%;
+
+  .lbl {
+    font-size: 14px;
+    font-weight: bold;
+    line-height: 18px;
+    margin: 0px;
+  }
+
+  .field {
+    border: 1px solid #dae1e3;
```

Comment:
Color literals like `#dae1e3` should only be used in variable declarations; they should be referred to via variable everywhere else.